src/app/tabs/run-tab.js

```diff
@@ -685,8 +701,12 @@ function settings (container, self) {
     fillAccountsList(el, self)
   }, 5000)
 
+  // setInterval(() => {
+  //   updateAccountBalances(container, self)
+  // }, 10000)
+
   setInterval(() => {
-    updateAccountBalances(container, self)
+    updateDeployedContractBalances('#runTabView .instance', self)
```

Comment:
if you remove `updateAccountBalances`, the balance of accounts are not going to be updated
maybe you should call both `updateDeployedContractBalances` and `updateAccountBalances`